charts/airbyte/templates/temporal/configmap.yaml

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
I don't know how easy this is to do, but in theory we could use https://github.com/temporalio/helm-charts, configured to use our dynamicconfig  + postgres.

Not blocking, but something for us to consider.